packages/react-checkbox/src/Checkbox.stories.tsx

```diff
@@ -1,30 +1,26 @@
 import * as React from 'react';
+import { Meta } from '@storybook/react';
 import { Checkbox } from './index';
 
-export const CheckboxVariations = () => (
-  <div style={{ display: 'flex', gap: '14px', flexDirection: 'column' }}>
-    <Checkbox>Simple Checkbox</Checkbox>
-    <Checkbox circular>Circular Checkbox</Checkbox>
-    <Checkbox labelPosition="before">Checkbox with label positioned before</Checkbox>
-    <Checkbox required>Required Checkbox</Checkbox>
-    <Checkbox size="large">Large Checkbox</Checkbox>
-    <Checkbox />
-    <Checkbox disabled />
-    <Checkbox disabled defaultChecked="mixed" />
-    <Checkbox disabled defaultChecked={true} />
-  </div>
-);
-
-export const CheckboxStates = () => (
-  <div style={{ display: 'flex', gap: '14px', flexDirection: 'column' }}>
-    <Checkbox>Unchecked Checkbox</Checkbox>
-    <Checkbox defaultChecked>Checked Checkbox</Checkbox>
-    <Checkbox defaultChecked="mixed">Mixed Checkbox</Checkbox>
-    <Checkbox disabled>Disabled Checkbox</Checkbox>
-  </div>
-);
+export { Default } from './Checkbox.Default.stories';
+export { Checked } from './Checkbox.Checked.stories';
+export { Mixed } from './Checkbox.Mixed.stories';
+export { Disabled } from './Checkbox.Disabled.stories';
+export { Large } from './Checkbox.Large.stories';
+export { LabelBefore } from './Checkbox.LabelBefore.stories';
+export { LabelWrapping } from './Checkbox.LabelWrapping.stories';
+export { Required } from './Checkbox.Required.stories';
+export { Circular } from './Checkbox.Circular.stories';
 
 export default {
   title: 'Components/Checkbox',
   component: Checkbox,
-};
+
+  decorators: [
+    Story => (
+      <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'start' }}>
```

Comment:
Might be good to also add a gap here (for more space between the checkboxes)

Reply:
The checkboxes have padding built in, and should not have a gap between them (per the figma spec).

Reply:
What I meant was having a gap between the checkboxes in examples (they can look kind of squished together otherwise)